group_vars/location_segen/networks.yml

```diff
@@ -0,0 +1,188 @@
+---
+# IPv4 10.31.6.0/24
+# IPv4 10.31.66.0/24
+# IPv6 2001:bf7:760:1000::/56
+
+# DHCP: 10.31.66.0/24
+# MGMT: 10.31.6.0/26
+# MESH: 10.31.6.64/26
+
+ipv6_prefix: "2001:bf7:760:1000::/56"
+
+networks:
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.6.0/26
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      segen-core: 1
+      segen-switch: 2
+
+      # bbb links
+      segen-saarbr: 10
+      segen-f2a: 11
+      segen-mabb: 12
+      segen-sw: 13
+      segen-bht: 14
+      segen-k12: 15
+      segen-no: 16
+      segen-wsee: 17
+      
+      # airos
+      segen-n-5ghz: 20
+      segen-o-5ghz: 21
+      segen-s-5ghz: 22
+      segen-w-5ghz: 23
+
+      # nf
+      segen-n-nf-2ghz: 24
+      segen-o-nf-2ghz: 25
+      segen-s-nf-2ghz: 26
+      segen-w-nf-2ghz: 27
+
+  - vid: 10
+    role: dhcp
+    prefix: 10.31.66.0/24
+    ipv6_subprefix: 0
+    untagged: true
+    inbound_filtering: false
+    enforce_client_isolation: false
+    assignments:
+      segen-core: 1
+
+  - vid: 101
+    role: mesh
+    name: mesh_saarbr
+    prefix: 10.31.6.64/32
+    ipv6_subprefix: -1
+    mesh_metric: 128
+    ptp: true
+
+  - vid: 102
+    role: mesh
+    name: mesh_f2a
+    prefix: 10.31.6.65/32
+    ipv6_subprefix: -2
+    mesh_metric: 1024
+    mesh_metric_lqm:
+      - default 0.3
+
+  - vid: 103
+    role: mesh
+    name: mesh_mabb
+    prefix: 10.31.6.66/32
+    ipv6_subprefix: -3
+
+  - vid: 104
+    role: mesh
+    name: mesh_sw
+    prefix: 10.31.6.67/32
+    ipv6_subprefix: -4
+
+  - vid: 105
+    role: mesh
+    name: mesh_bht
+    prefix: 10.31.6.68/32
+    ipv6_subprefix: -5
+
+  - vid: 106
+    role: mesh
+    name: mesh_k12
+    prefix: 10.31.6.69/32
+    ipv6_subprefix: -6
+
+  - vid: 107
+    role: mesh
+    name: mesh_no
+    prefix: 10.31.6.70/32
+    ipv6_subprefix: -7
+
+  - vid: 108
+    role: mesh
+    name: mesh_wsee
+    prefix: 10.31.6.71/32
+    ipv6_subprefix: -8
+
+
+  # 11s mesh 2ghz
+  - vid: 201
+    role: mesh
+    name: mesh_11s_n2
+    prefix: 10.31.6.72/32
+    ipv6_subprefix: -9
+    mesh_metric: 1024
+    mesh_ap: segen-n-nf-2ghz
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+    mac_override: f0:9f:c2:0f:79:31
```

Comment:
why are you doing this?  dont know this feature. Mac override works differently:
See following links for reference
https://github.com/freifunk-berlin/bbb-configs/blob/29a001b233854b03cdf0a16724b971a706aab8b3/roles/cfg_openwrt/templates/common/config/network.j2#L80
https://github.com/freifunk-berlin/bbb-configs/blob/29a001b233854b03cdf0a16724b971a706aab8b3/host_vars/magda-ap1/base.yml#L7

Reply:
The seperate MAC addresses prevent warning messages about packets coming in on an interface with the same MAC address.  This would happen when two 802.11s devices on the same core system see each other.

The end result is
```
config interface 'mesh_11s_n2'
	option device 'switch0.201'
	option macaddr 'f0:9f:c2:0f:79:31'
	option proto 'static'
	option ipaddr '10.31.6.72/32'
	option ip6addr '2001:bf7:760:10f7::1/128'

config interface 'mesh_11s_o2'
	option device 'switch0.202'
	option macaddr 'f0:9f:c2:0f:79:32'
	option proto 'static'
	option ipaddr '10.31.6.73/32'
	option ip6addr '2001:bf7:760:10f6::1/128'

config interface 'mesh_11s_s2'
	option device 'switch0.203'
	option macaddr 'f0:9f:c2:0f:79:33'
	option proto 'static'
	option ipaddr '10.31.6.74/32'
	option ip6addr '2001:bf7:760:10f5::1/128'

config interface 'mesh_11s_w2'
	option device 'switch0.204'
	option macaddr 'f0:9f:c2:0f:79:34'
	option proto 'static'
	option ipaddr '10.31.6.75/32'
	option ip6addr '2001:bf7:760:10f4::1/128'
```
Which I think is the proper way for it to look.

Reply:
This issue is already solved https://github.com/freifunk-berlin/bbb-configs/issues/119

Reply:
I have no idea if this works and I don't not have the capacity at the moment to test it.

Reply:
As discussed with all participants: Test if the fix #119 actually works and remove the overrides if it does. If the warning messages start showing up, keep the overrides and create an issue to fix the underlying problem in general.